precompile/contracts/feemanager/contract_test.go

```diff
@@ -248,6 +250,78 @@ var (
 			ReadOnly:    false,
 			ExpectedErr: vmerrs.ErrOutOfGas.Error(),
 		},
+		"set config doesn't log before DUpgrade": {
```

Comment:
same comment from other event PRs, doesn't creating a before and after + modifying the previously existing test create a duplicate of the same case?

Reply:
I moved after hook to existing test.